e2e/playwright/events/edit_event.spec.ts

```diff
@@ -0,0 +1,205 @@
+        await login(page);
+        await waitForPageLoad.planning(page);
+        await subnav.createEvent();
+        await editor.waitTillOpen();
+    });
+
+    test('can create an Event', async () => {
+        await list.expectEmpty();
+        await editor.expectItemType();
+        await workqueue.expectTitle(0, 'Untitled*');
+
+        await editor.openAllToggleBoxes();
+        await editor.type(event);
+        await editor.expect(expectedEvent);
+        await editor.waitForAutosave();
+
+        await workqueue.expectTitle(0, 'slugline of the event*');
+        await editor.createButton.click();
+        await list.expectItemCount(1);
+        await list.expectItemText(0, 'slugline of the event');
+        await workqueue.expectTitle(0, 'slugline of the event');
+    });
+
+    test('can create a Recurring Event', async ({page}) => {
+        await setupPlanningPublishing(page.request);
+        await list.expectEmpty();
+        await editor.expectItemType();
+
+        // TODO(petr): make recurring events work with all day events
+        event = {
+            ...event,
+            'dates.recurring.enable': true,
+            'dates.recurring.until': moment().add(2, 'day')
+                .format(CLIENT_FORMAT),
+            slugline: 'slugline of the recurring event',
+            name: 'name of the recurring event',
+        };
+        expectedEvent = {
+            ...expectedEvent,
+            'dates.recurring.enable': true,
+            'dates.recurring.until': moment().add(2, 'day')
+                .format(CLIENT_FORMAT),
+            slugline: 'slugline of the recurring event',
+            name: 'name of the recurring event',
+        };
+
+        await editor.openAllToggleBoxes();
+        await editor.type(event);
+        await editor.expect(expectedEvent);
+
+        await editor.waitForAutosave();
+        await editor.createButton.click();
+        await list.expectItemCount(2);
+        await list.expectItemText(0, 'slugline of the recurring event');
+        await list.expectItemText(1, 'slugline of the recurring event');
+
+        // Test cancelling the Post modal
+        await editor.postButton.click();
+        await modal.waitTillOpen(30000);
+        await modal.getFooterButton('Cancel')
+            .click();
+        await modal.waitTillClosed();
+
+        // Test posting the series
+        await editor.postButton.click();
+        await modal.waitTillOpen(30000);
+        await modal.getFooterButton('Post')
+            .click();
+        await modal.waitTillClosed();
+
+        // Make sure both recurring Events now have the 'Scheduled' badge
+        await expect(
+            list.item(0).locator('.label--success'),
+        ).toContainText('Scheduled');
+        await expect(
+            list.item(1).locator('.label--success'),
+        ).toContainText('Scheduled');
+    });
+
+    test('SDESK-5982: Post updates the initial values', async ({page}) => {
```

Comment:
probably we don't need the ticket number here
```suggestion
    test('Post updates the initial values', async ({page}) => {
```